src/api/services/usersService.ts

```diff
@@ -20,15 +20,9 @@ export interface SelfResendInviteRequest extends Request {
 
 export type UserWithIsApprover = User & { isApprover: boolean };
 
-export const findUserByEmail = async (email: string) => {
```

Comment:
I don't think the previous implementation was causing issues, but it had some duplicate data which we can remove.